CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,4 @@
 - Update Howtos with the last changes of the IoT Agent library (#48).
 - Add test cases to check first observation of a value (#54).
 - Update to IoTAgent Library 0.10.0 and LWM2M Library 0.5.0.
+- FEATURE update node version to 4.8.4
```

Comment:
I think this has been just merged as part of PR #92... maybe this PR's branch should be upgrades with master so this change would not be shown?

In addition, maybe above line should mention the new iotagent library version, i.e. 0.10.0 -> 2.4.x (although in packages.json dependency is expeses at a git URL, 2.4.x is the newer version name at the present moment)

Reply:
Fixed in c86cf5c10739653161112be9133ff978bb8c386c

Reply:
The part about "FEATURE update node version to 4.8.4" out is ok but... what about "Update to IoTAgent Library 0.10.0 and LWM2M Library 0.5.0". After this PR IoTAgent Library is no longer 0.10.0...

Reply:
Fixed in f4907371379b3f5724e88f2408d0e69c656ee5a4